### PR TITLE
Feat: create redis client for identity service

### PR DESCRIPTION
**Pull request by Nhsdkk**
1) Create redis client

### CODE REVIEW

File: identity_service/main.py [hunk 1]
```diff
@@ -16,7 +17,8 @@ async def serve():
         IdentityServiceImpl(), server
     )
     server.add_insecure_port("0.0.0.0:8080")
-    await Db().connect()
+    await PostgresClient().connect()
+    RedisClient()
```
Comment (MaxPlays35) [inline identity_service/main.py:21]: It's doubtful, but okay.

**Review by MaxPlays35 — CHANGES_REQUESTED** [review]

**Review by MaxPlays35 — APPROVED** [review]

File: identity_service/main.py (hunk 1 above)
Comment (Nhsdkk) [reply identity_service/main.py:21]: Here i just renamed the class. Seems fine to me

File: identity_service/db/redis_client.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+@singleton
+class RedisClient:
+    """
+    Data class that stores user information
+
+    Attributes
+    ----------
+    db : redis.Redis
+        Redis client
+
```
Comment (ZixMai) [inline identity_service/db/redis_client.py]: **DELETE THIS EMPTY STRING** @Nhsdkk

**Review by ZixMai — CHANGES_REQUESTED** [review]

File: identity_service/db/redis_client.py (hunk 2 above)
Comment (Nhsdkk) [reply identity_service/db/redis_client.py]: fixed

**Review by ZixMai — APPROVED** [review]

**Pull request merged**